script/Makefile.mk

```diff
@@ -42,26 +41,20 @@ else
 endif
 
 #
-# If PROJECT is defined, compute default GCR name (unless explicitly defined).
+# If REGISTRY is defined, prepare to push images.
 #
-ifneq ($(origin PROJECT),undefined)
-GCR ?= gcr.io/$(PROJECT)/catalog
-endif
-
-#
-# If GCR is defined, prepare to push images.
-#
-ifeq ($(origin GCR),undefined)
+ifeq ($(origin REGISTRY),undefined)
```

Comment:
I like that behavior. I would worry if unqualified 'make push' actually tried to push somewhere as that could cause unintended overwriting of things. What @krancour proposes seems like perfect thing - it is explicit but very usable, and unbiased.